tests/layer_tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
 -c ../constraints.txt
 # paddlepaddle #  ticket 95904
 numpy
-onnxruntime
+onnxruntime >= 1.18.0
```

Comment:
Please limit the upper bound for the Python3.9 to 1.19.2 as well according to https://onnxruntime.ai/roadmap